### ISSUE TITLE
MultiJson::DecodeError

### ISSUE THREAD

**Issue by bullfight** [brightcove-archive/zencoder_fetcher#4]
After Having run `gem update --system` to deal with some other issues, I can no longer successfully run zencoder fetcher.

I am running

```
$ruby -v
ruby 1.9.2p290 (2011-07-09 revision 32553) [x86_64-darwin10.8.0]

$gem-v
1.8.10
```

I am getting the following error

```
Checking Zencoder for Notifications
{}
MultiJson::DecodeError
743: unexpected token at 'Thanks, Zencoder!'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/json-1.6.0/lib/json/common.rb:148:in `parse'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/json-1.6.0/lib/json/common.rb:148:in `parse'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/multi_json-1.0.3/lib/multi_json/engines/json_gem.rb:13:in `decode'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/multi_json-1.0.3/lib/multi_json.rb:65:in `decode'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/parser.rb:116:in `json'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/parser.rb:136:in `parse_supported_format'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/parser.rb:103:in `parse'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/parser.rb:66:in `call'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/request.rb:221:in `parse_response'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/request.rb:193:in `handle_response'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty/request.rb:75:in `perform'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty.rb:391:in `perform_request'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty.rb:359:in `post'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/httparty-0.8.0/lib/httparty.rb:427:in `post'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/lib/zencoder_fetcher.rb:40:in `block in request'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/lib/zencoder_fetcher.rb:35:in `each'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/lib/zencoder_fetcher.rb:35:in `request'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/bin/zencoder_fetcher:64:in `block in <top (required)>'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/bin/zencoder_fetcher:61:in `loop'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/gems/zencoder-fetcher-0.2.2/bin/zencoder_fetcher:61:in `<top (required)>'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/bin/zencoder_fetcher:19:in `load'
/Users/pschmitz/.rvm/gems/ruby-1.9.2-p290@goswim/bin/zencoder_fetcher:19:in `<main>'
rake aborted!
```

**Comment by bullfight** [comment]
This was solved by upgrading to 0.2.3.  I needed to remove the dependency on `"activesupport", "~> 2.3.11"`
As in this commit
https://github.com/zencoder/zencoder-fetcher/commit/1b4d966aaa485a543f3df8e796d4abd7d1bbf7e1

**Comment by chriswarren** [comment]
That'll work if you're using JSON, but it'll break XML support because we're including ActiveSupport for its to_xml method, which was moved in Rails 3. I'll see what I can find to use in its place so we can try to avoid this.

**Comment by chriswarren** [comment]
This should be resolved in 0.2.4.
